### PR TITLE
Added question API calls to frontend.

### PR DESCRIPTION
**Pull request by kylejustbenson**
Added API calls to frontend for submitting a question to the server and
listing all questions associated with a board.

### CODE REVIEW

File: src/main/java/org/query/RequestHandlers.java [hunk 1]
```diff
@@ -52,8 +52,8 @@ public String getDashboard(@PathVariable("username") String username) {
   }
 
   @RequestMapping(value="/q", method=RequestMethod.GET)
-  public List<Question> getQuestions(@RequestParam String qid) {
-    List<Question> questions = Database.getAllQuestions(Long.valueOf(qid));
+  public List<Question> getQuestions(@RequestParam("bid") String bid) {
```
Comment (nathanmilot) [inline src/main/java/org/query/RequestHandlers.java]: This doesn't seem to be parsing correctly. Is it supposed to be yet? Or are you just setting up functionality?

Comment (nathanmilot) [reply src/main/java/org/query/RequestHandlers.java]: Sorry, line 56 is the issue

Comment (kylejustbenson) [reply src/main/java/org/query/RequestHandlers.java]: It should be working. I compiled and tested it on my machine. What is the issue specifically?

Comment (nathanmilot) [reply src/main/java/org/query/RequestHandlers.java]: so it says it's trying to parse `home` as long. I'm messing around with it right now to make sure there isn't something weird going on on my setup.

Comment (kylejustbenson) [reply src/main/java/org/query/RequestHandlers.java]: As we discussed on discord, the "/home" mapping is just a placeholder and the real calls will be to "/b/{bid}" with a long to parse. I have some issues with using just the bid as the url, but that's a bridge we can cross after the prototype.

**Pull request merged**